Common/src/main/resources/defaultresources/excavated_variants/excavated_variants/variants/minecraft.json5

```diff
@@ -160,10 +160,6 @@
   provided_ores: [
     {
       id: 'coal_ore',
```

Comment:
Uh, what happened to this file?

Reply:
I don't even remember lmao.

Reply:
This PR shouldn't touch this file if you're not changing something in it

Reply:
Yeah, I think I removed the spacing from the multi-line comment or something. It's been months, so I don't remember - but since it's showing no changes, it would've been something minor like that.